specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.1-preview.2/FormRecognizer.json

```diff
@@ -775,11 +778,33 @@
         "description": "Extract text and layout information from a given document. The input document must be of one of the supported content types - 'application/pdf', 'image/jpeg', 'image/png', 'image/tiff' or 'image/bmp'. Alternatively, use 'application/json' type to specify the location (Uri or local path) of the document to be analyzed.",
         "operationId": "AnalyzeLayoutAsync",
         "consumes": [
+          "application/pdf",
+          "application/json",
+          "image/jpeg",
+          "image/png",
+          "image/tiff",
+          "image/bmp"
         ],
         "produces": [
           "application/json"
         ],
         "parameters": [
+          {
+            "name": "language",
+            "in": "query",
+            "description": "The BCP-47 language code of the text in the document. Currently, only English ('en'), Dutch (‘nl’), French (‘fr’), German (‘de’), Italian (‘it’), Portuguese (‘pt'), simplified Chinese ('zh-Hans') and Spanish ('es') are supported (print – nine languages and handwritten – English only). Layout supports auto language identification and multilanguage documents, so only provide a language code if you would like to force the documented to be processed as that specific language.",
+            "type": "string",
```

Comment:
Let's reference #/definitions/Language instead.  We need to expand the set of values there though.

Reply:
bit tricky, suspect swagger syntax limitation, in parameters schema is supported for parameters in "body"